src/main/kotlin/kotlinx/kover/api/KoverConfig.kt

```diff
@@ -240,11 +240,11 @@ public open class KoverProjectHtmlConfig @Inject constructor(private val objects
 
 
 public open class KoverMergedConfig @Inject constructor(objects: ObjectFactory) {
-    internal var isEnabled: Property<Boolean> = objects.property(Boolean::class.java)
-    internal val filters: KoverMergedFilters = objects.newInstance(KoverMergedFilters::class.java, objects)
-    internal val xmlReport: KoverMergedXmlConfig = objects.newInstance(KoverMergedXmlConfig::class.java, objects)
-    internal val htmlReport: KoverMergedHtmlConfig = objects.newInstance(KoverMergedHtmlConfig::class.java, objects)
-    internal val verify: KoverVerifyConfig = objects.newInstance(KoverVerifyConfig::class.java, objects)
```

Comment:
Because the instances being created here have constructors annotated with `@Inject`, there's no need to pass in `objects` manually.